File: A/ABC333.cpp
#include <iostream>

using namespace std;

int main() {
    int a, b;
    cin >> a >> b;
    
    bool flag = false;
    for(int i=1; i <= 3;i++){
        if(a*b*i%2 == 1){
            flag = true;
        }
    }

    if(flag){
        cout << "Yes" << endl;
    } else {
        cout << "No" << endl;
    }
}

File: rust/src/A/abc333.rs
fn main() {
    let mut flag: bool = false;
    proconio::input! {
        a: i32,
        b: i32,
    }

    for i in 0..3 {
        if a*b*i%2 == 1 {
            flag = true;
        }
    }

    if flag {
        println!("Yes")
    } else {
        println!("No")
    }
}